images/ebs-warmup/warmup-steps.sh

```diff
@@ -53,7 +54,16 @@ cleanup() {
 trap cleanup EXIT
 
 operation=none
+exit_on_corruption=false
 bg_works=""
+
+for arg in "$@"; do
+    case $arg in
+        --exit-on-corruption) exit_on_corruption=true
```

Comment:
We may also need to match this flag before line 79 or the script may exit due to the unrecognized flag (Maybe parse this flag in the `while` loop will be fine?):

https://github.com/pingcap/tidb-operator/pull/5622/files#diff-113f3c9592cd00b3ff52744edb177373f178c3c7a0ed08607af5558a7f54d76aR79-R80

Reply:
ah you're right, this will exit in the next while loop if provided .

however the while loop below does both arg parsing and does the work of WAL checking, so parsing there requires explicit ordering of `--exit-on-corruption` flag prior to any other args. while we do specify the correct ordering when generating warmup args, would prefer to not rely on it.

will keep the parsing here and then just no-op on the `exit-on-corruption` flag in the while loop below